### PR TITLE
Letter D input param changes

### PR DESCRIPTION
**Pull request by austinborn**
## Changes

- changed input parameters to new interface
- updated schemas/env.json with description and title

## Quality Assurance

- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [x] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [x] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

File: packages/sources/deribit/src/index.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { expose } from '@chainlink/ea-bootstrap'
-import { makeExecute } from './adapter'
+import { makeExecute, endpointSelector } from './adapter'
 import { makeConfig, NAME } from './config'
+import * as types from './endpoint'
```
Comment (justinkaseman) [inline packages/sources/deribit/src/index.ts:4]: :raised_hands:

File: packages/sources/dxfeed/src/endpoint/price.ts [hunk 2]
```diff
@@ -14,7 +14,18 @@ export const batchablePropertyPath = [{ name: 'base', limit: 120 }]
...
 
 export const inputParameters: InputParameters = {
-  base: ['base', 'from', 'coin', 'market'],
+  base: {
+    required: true,
+    aliases: ['from', 'coin', 'market'],
+    description: 'The symbol of the currency to query',
+    type: 'string',
+  },
+  overrides: {
```
Comment (justinkaseman) [inline packages/sources/dxfeed/src/endpoint/price.ts:23]: This one is a base input parameter that is shared across every EA.

Probably don't need it here.

Comment (austinborn) [reply packages/sources/dxfeed/src/endpoint/price.ts:23]: Does this apply to these other properties as well?
```
const excludableInternalAdapterRequestProperties = [
  'resultPath',
  'overrides',
  'tokenOverrides',
  'includes',
]

Comment (justinkaseman) [reply packages/sources/dxfeed/src/endpoint/price.ts:23]: Yeah definitely. We should convert those to use the new input parameter syntax to give them descriptions.

Comment (austinborn) [reply packages/sources/dxfeed/src/endpoint/price.ts:23]: Ticket: https://app.shortcut.com/chainlinklabs/story/25085/add-references-to-shared-inputs-in-each-ea-readme
For now, I'll just remove these properties from every endpoint's input parameters.

**Review by justinkaseman — APPROVED** [review]

**Pull request merged**